src/components/domain/transactions/searchField/TransactionsSearchField.tsx

```diff
@@ -0,0 +1,27 @@
+import { Group } from 'react-aria-components'
+import { Search } from 'lucide-react'
+import { MinimalSearchField } from '../../../ui/SearchField/MinimalSearchField'
+import { VStack } from '../../../ui/Stack/Stack'
+
+const CLASS_NAME = 'Layer__TransactionsSearchField'
+
+type TransactionsSearchFieldProps = {
+  value: string
+  slot?: string
+  onChange: (value: string) => void
+}
+
+export function TransactionsSearchField({ slot, ...restProps }: TransactionsSearchFieldProps) {
+  return (
+    <Group slot={slot} className={CLASS_NAME}>
+      <VStack slot='icon' align='center' justify='center'>
+        <Search size={14} />
+      </VStack>
+      <MinimalSearchField
+        {...restProps}
+        slot='search'
+        placeholder='Search transactions'
```

Comment:
Before we role this out... do you have a product opinion on whether we should leave this search as a filter on description / merchant name vs. turning it into an omni-search where maybe it also searches by account name, amount, etc? 
Non-trivial feature ofc.